observables/car-headlights.tag

```diff
@@ -0,0 +1,24 @@
+<car-headlights>
+  <div class="light" if={ engineOn }></div>
+  <div class="light" if={ engineOn }></div>
+
+  <script>
+    var _this = this
```

Comment:
Could you change `_this` to `self`?
The properties prefixed with `_` is reserved for the private use in Riot, kind of.